---
title: Datasets and Research Resources
permalink: datasets
---
## 
![Dashboard](/assets/imags/dashboard.png){: .align-left width="500px"}
**Global biogeography of *Ralstonia* pathogens:** Seeking to unify the extensive literature on Ralstonia populations around the globe, we have an ongoing literature meta-analysis of *Ralstonia solanacearum* isolations (over 10,000 strains from over 300 papers). We publish updates to the *Ralstonia* isolation dataset as a ["living" BioRxiv pre-print (Lowe-Power et al.).](https://www.biorxiv.org/content/10.1101/2020.07.13.189936v5). In collaboration with Dr. Tessa Pierce-Ward, also adapted a web-based, interactive Dashboard that allows users to investigate patterns of *Ralstonia* biogeography, phylogeny, and host-of-isolation. Dashboard: https://ralstoniadashboard.shinyapps.io/RalstoniaWiltDashboard/  Our YouTube channel (linked below) has tutorials and case studies to demonstrate how to use the Dashboard to understand Ralstonia. 

## Lowe-Power Lab YouTube Channel
[Link to the channel](https://www.youtube.com/channel/UC0x1gbM48khnawBZ8AfAgxA)

[Seminar on Ralstonia biogeography and host range]()
* [Companion pre-print/database](https://www.biorxiv.org/content/10.1101/2020.07.13.189936v4)

[Seminar on the physiological mechanisms that underly wilt symptoms during Ralstonia disease]()
* [Companion paper](https://apsjournals.apsnet.org/doi/full/10.1094/PHYTOFR-06-21-0041-R)

## FigShare
[On FigShare, we release cartoons and raw data from papers.](https://figshare.com/authors/Tiffany_Lowe-Power/6561146) 
Most data is released with a CC-BY license that only requires attribution and does not restrict any reuse of the data. 

## KBase 
We use KBase to generate organized datasets for comparative evolutionary genomics studies. We also have created public narratives that allow anyone to re-analyze Ralstonia RNAseq data from SRA. 

We currate a list of useful KBase narratives on a page of our Protocols GitHub:
[https://github.com/lowepowerlab/protocols/blob/main/kbase.md](https://github.com/lowepowerlab/protocols/blob/main/kbase.md) 

## Interactive files to understand genetic organization of *Ralstonia's* vgr-linked toxin/antitoxin clusters
These tools were built as part of our study: 

* A pangenomic atlas reveals that eco-evolutionary dynamics shape plant pathogen type VI secretion systems
Nathalie Aoun, Stratton J. Georgoulis, Jason K. Avalos, Kimberly J. Grulla, Kasey Miqueo, Cloe Tom, Tiffany M. Lowe-Power
bioRxiv 2023.09.05.556054; doi: [https://doi.org/10.1101/2023.09.05.556054](https://doi.org/10.1101/2023.09.05.556054)

* There are not any recording of seminars on this study, but we have a slide deck available: [https://figshare.com/articles/presentation/The_ecology_and_evolution_of_Ralstonia_s_toxin-secreting_molecular_weapon_T6SS_2023-08-18_ICPP_talk/23992947](https://figshare.com/articles/presentation/The_ecology_and_evolution_of_Ralstonia_s_toxin-secreting_molecular_weapon_T6SS_2023-08-18_ICPP_talk/23992947)

### Methods: 

The interative synteny analysis files were created using Clinker. [The documentation for Clinker can be found here](https://github.com/gamcil/clinker)

**Reference:** clinker & clustermap.js: Automatic generation of gene cluster comparison figures. Gilchrist, C.L.M., Chooi, Y.-H., 2020. Bioinformatics.

### Synteny files organized by aux cluster 

If you would like to re-analyze the files used in this dataset, the inputs are located in a repository on our FigShare (see link above)

* [Interative HTML for aux1 (3 examples)](https://lowepowerlab.github.io/assets/clinker/vgr_aux/aux_types/aux1_Fig.html)
* [Interative HTML for aux2 (4 examples)](https://lowepowerlab.github.io/assets/clinker/vgr_aux/aux_types/aux2_Fig.html)
* [Interative HTML for aux3 (8 examples)](https://lowepowerlab.github.io/assets/clinker/vgr_aux/aux_types/aux3_Fig.html)
* [Interative HTML for aux4 (7 examples)](https://lowepowerlab.github.io/assets/clinker/vgr_aux/aux_types/aux4_Fig.html)
* [Interative HTML for aux5 (6 examples)](https://lowepowerlab.github.io/assets/clinker/vgr_aux/aux_types/aux5_Fig.html)
* [Interative HTML for aux6 (9 examples)](https://lowepowerlab.github.io/assets/clinker/vgr_aux/aux_types/aux6_Fig.html)
* [Interative HTML for aux7 (11 examples)](https://lowepowerlab.github.io/assets/clinker/vgr_aux/aux_types/aux7_Fig.html)
* [Interative HTML for aux8 (6 examples)](https://lowepowerlab.github.io/assets/clinker/vgr_aux/aux_types/aux8_Fig.html)
* [Interative HTML for aux9 (8 examples)](https://lowepowerlab.github.io/assets/clinker/vgr_aux/aux_types/aux9_Fig.html)
* [Interative HTML for aux10 (5 examples)](https://lowepowerlab.github.io/assets/clinker/vgr_aux/aux_types/aux10_Fig.html)
* [Interative HTML for aux11 (3 examples)](https://lowepowerlab.github.io/assets/clinker/vgr_aux/aux_types/aux11_Fig.html)
* [Interative HTML for aux12 (4 examples)](https://lowepowerlab.github.io/assets/clinker/vgr_aux/aux_types/aux12_Fig.html)
* [Interative HTML for aux13 (3 examples)](https://lowepowerlab.github.io/assets/clinker/vgr_aux/aux_types/aux13_Fig.html)
* [Interative HTML for aux14-only (4 examples), aux20-only (3 examples), and combination aux14/aux20 (5 examples)](https://lowepowerlab.github.io/assets/clinker/vgr_aux/aux_types/aux14_aux20_Fig.html)
* [Interative HTML for aux15 (4 examples) and aux18 (5 examples)](https://lowepowerlab.github.io/assets/clinker/vgr_aux/aux_types/aux15_aux18_Fig.html)
* [Interative HTML for aux16 (5 examples)](https://lowepowerlab.github.io/assets/clinker/vgr_aux/aux_types/aux16_Fig.html)
* [Interative HTML for aux17 (8 examples)](https://lowepowerlab.github.io/assets/clinker/vgr_aux/aux_types/aux17_Fig.html)
* [Interative HTML for aux19 (2 examples)](https://lowepowerlab.github.io/assets/clinker/vgr_aux/aux_types/aux19_Fig.html)
* [Interative HTML for aux21 (5 examples)](https://lowepowerlab.github.io/assets/clinker/vgr_aux/aux_types/aux21_Fig.html)
* [Interative HTML for aux at tssK: aux10 (2 examples), aux17 (2 examples), aux22 (5 examples), aux23 (2 examples), aux24 (1 example) and a combination aux10/aux24 (1 example)](https://lowepowerlab.github.io/assets/clinker/vgr_aux/aux_types/aux22_23_24_10_17_Fig.html)
* [Interative HTML for aux25 (3 examples)](https://lowepowerlab.github.io/assets/clinker/vgr_aux/aux_types/aux25_Fig.html)



# Plasmid Maps

These plasmids are described in the linked publications. All plasmid maps are provided as read-only Benchling links. If you prefer a different cloning software, Benchlings allows exports as genbank files (.gb). If you find a broken link, or you're looking for a vector sequence that I've overlooked, please [email me](mailto:tiff.lowe@gmail.com)

## 2018
{: .text-center}

Plasmids from: *Metabolomics of tomato xylem sap during bacterial wilt reveals Ralstonia solanacearum produces abundant putrescine, a metabolite that accelerates wilt disease.*
[Download PDF](/assets/papers/pub_xylem_metabo_put.pdf){: .btn .btn--info .btn--small}

[Benchling: pKO-speC::Sm](https://benchling.com/s/seq-7CZyOX1rZF5NYhvGvVcQ){: .btn .btn--info .btn--success}
[Benchling: pRCTspeC_com](https://benchling.com/s/seq-NKO3SxZWZp4gFhLDKveJ){: .btn .btn--info .btn--success}
[Benchling: pUFR80-hrcC](https://benchling.com/s/seq-jLqhQ35GbJ4PQV8cZeRg){: .btn .btn--info .btn--success}

## 2017
{: .text-center}

Plasmids from: *Plant-like bacterial expansins play contrasting roles in two tomato vascular pathogens.* [Download PDF](/assets/papers/pub_clavi_ralsto_expansins.pdf){: .btn .btn--info .btn--small}

[Benchling: pUFR80-RSc0818KO](https://benchling.com/s/seq-bsZQQpr8UoQDv8qehPZn){: .btn .btn--info .btn--success}
[Benchling: pMiniTn7-RSc0818comp](https://benchling.com/s/seq-Ghja0tFUNOHYhPYrafjT){: .btn .btn--info .btn--success}
[Benchling: pHN_RsExpA](https://benchling.com/s/seq-Gezq36zTRcyptq7Vvh3t){: .btn .btn--info .btn--primary}

## 2016
{: .text-center}

Plasmids from: *Degradation of the plant defense signal salicylic acid protects *Ralstonia solanacearum* from toxicity and enhances virulence on tobacco.* [Download PDF](/assets/papers/pub_ralstonia_salicylic_deg.pdf){: .btn .btn--info .btn--small}

[Benchling: pUFR80-KOnagGH](https://benchling.com/s/seq-6Xfj3n9yAmHix8q0Zuek){: .btn .btn--info .btn--success}
[Benchling: pST-KOnagAaGHAbIKL](https://benchling.com/s/seq-oMgGdjfCB9bbxNvrntFZ){: .btn .btn--info .btn--success}
[Benchling: pRCGnagGHcomp](https://benchling.com/s/seq-Kbz9uizoeG0FqqkQOOeL){: .btn .btn--info .btn--success}
[Benchling: pMiniTn7-nagAaGHAbIKL-comp](https://benchling.com/s/seq-3ccZp1gI3dEIuMTMS87F){: .btn .btn--info .btn--success}
[Benchling: pMiniTn7-KRNP](https://benchling.com/s/seq-ERBQTjoZl4zWM9WsP2XL){: .btn .btn--info .btn--success}

Plasmids from: *Functional identification of putrescine C- and N-hydroxylases.*
[Download PDF](/assets/papers/pub_hydroxy_putrescine.pdf){: .btn .btn--info .btn--small}

[Benchling: pUFR80-2HPut](https://benchling.com/s/seq-rX7hpZHENqxwTGHGKoFv){: .btn .btn--info .btn--success}

## 2015
{: .text-center}

Plasmids from: *Hydroxycinnamic acid degradation, a broadly conserved trait, protects Ralstonia solanacearum from chemical plant defenses and contributes to root colonization and virulence.*
[Download PDF](/assets/papers/pub_ralstonia_hca_degradation.pdf){: .btn .btn--info .btn--small}

[Benchling: pUFR80-fcsKO](https://benchling.com/s/yJIsar7R){: .btn .btn--info .btn--success}
[Benchling: pMiniTn7FcsComp](https://benchling.com/s/0yveOCu9){: .btn .btn--info .btn--success}

## Parental Plasmids

Most of the plasmids I use frequently (e.g. pST-Blue, pUC18miniTn7t-Gm, & pRC* series for *Ralstonia solanacearum* GMI1000) are available online (PubMed or via manufacturers). However, I had to track down or reconstruct a few plasmid sequences.  Those are listed below with original citations.

### pUFR80 (Kan/sacB vector for markerless deletions)

Casta√±eda A, Reddy JD, El-Yacoubi B, Gabriel DW. 2005. Mutagenesis of all eight avr genes in Xanthomonas campestris pv. campestris had no detected effect on pathogenicity, but one avr gene affected race specificity. Mol Plant-Microbe Interact 18:1306-1317. 

[Benchling: pUFR80](https://benchling.com/s/SGEEU7){: .btn .btn--info .btn--success}

### pAX1-Cm, pAX1-Em, & pAX1-Gm (Chromosomal complementation of Xylella Temecula1

Matsumoto, Young, and Igo. 2009. Chromosome-based genetic complementation system for Xylella fastidiosa. Appl Env Micro 75:1679-1687. 

[Benchling: pAX1-Cm](https://benchling.com/s/seq-P5ieKhi83h8dF3sZVh8m){: .btn .btn--info .btn--success}
[Benchling: pAX1-Em](https://benchling.com/s/seq-lKAzFa9aV3hdEMUwwtTM){: .btn .btn--info .btn--success}
[Benchling: pAX1-Gm](https://benchling.com/s/seq-9C532mruhQPR3jndxFrB){: .btn .btn--info .btn--success}
